lib/build.sh

```diff
@@ -20,3 +20,10 @@ check_tsc_binary() {
 
   [[ -f "$build_dir/node_modules/typescript/bin/tsc" ]]
 }
+
+check_tsc() {
```

Comment:
let's change this method name to something like "compile_source_code" or "build_ts_app" so it's more descriptive of what the method is doing.